charts/kube-prometheus-stack.yaml

```diff
@@ -12,7 +12,8 @@ commonLabels:
 alertmanager:
   alertmanagerSpec:
     image:
-      repository: '__image__(alertmanager)'
+      registry: '__var__(repo.registry_endpoint)'
+      repository: '__image_no_reg__(alertmanager)'
```

Comment:
Sad, but yes it's needed

But cannot we set this registry only once in `global` ?

Reply:
Right will do, found out too late about this one 😇

Reply:
Argh no I can't do that because the kube-state-metrics subchart doesn't use this value correctly:

```
image: "{{ .Values.global.imageRegistry | default .Values.image.repository }}:{{ .Values.image.tag | ...
```

:sob: 

I'll submit a PR over there to have it fixed